languages/multi-language/README.md

```diff
@@ -0,0 +1,7 @@
+# Multi-Language module
+
+This module allows parsing multiple languages in one run of JPlag. The parsing will be delegated to a different language-module per file.
+
+This does not allow comparing code from different languages, but it can be used to compare multiple languages in the same run.
```

Comment:
Let's be even more precise here to avoid confusion. I suggest this:
"This does not entail comparing implementations of the same functionality in different languages to each other, but allows comparing submissions that use more than one language."